src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.String.cs

```diff
@@ -41,6 +41,29 @@ public static partial class JsonSerializer
                 throw new ArgumentNullException(nameof(json));
             }
 
+            return Deserialize<TValue>(json.AsSpan(), typeof(TValue), options);
+        }
+
+        /// <summary>
+        /// Parse the text representing a single JSON value into a <typeparamref name="TValue"/>.
+        /// </summary>
+        /// <returns>A <typeparamref name="TValue"/> representation of the JSON value.</returns>
+        /// <param name="json">JSON text to parse.</param>
+        /// <param name="options">Options to control the behavior during parsing.</param>
+        /// <exception cref="JsonException">
+        /// Thrown when the JSON is invalid,
+        /// <typeparamref name="TValue"/> is not compatible with the JSON,
+        /// or when there is remaining data in the Stream.
```

Comment:
This overload doesn't take a stream. Doc comment needs to be updated.

I recommend using the content from the official msft docs that have gone through content review to seed the new API docs rather than copying from an overload to avoid slight discrepancies.
https://docs.microsoft.com/en-us/dotnet/api/system.text.json.jsonserializer.deserialize?view=netcore-3.1#System_Text_Json_JsonSerializer_Deserialize__1_System_String_System_Text_Json_JsonSerializerOptions_

https://github.com/dotnet/dotnet-api-docs/blob/80088021b2151fd5af3cda6e759ec5e548c372c0/xml/System.Text.Json/JsonSerializer.xml#L281-L348